flowzone.yml

```diff
@@ -3317,12 +3317,18 @@ jobs:
           provenance: false
 
       # save image to file before running compose tests to avoid tainting the published image
-      - name: Save image to file
+      - name: Save image to file scan it and archive it
         if: needs.is_docker.outputs.docker_publish_matrix != ''
         run: |
           docker save ${{ join(fromJSON(steps.test_meta.outputs.json).tags,' ') }} -o ${DOCKER_TAR}
+          
+          docker run --rm -v ${DOCKER_TAR}:/tmp/image.tar \
+          trufflesecurity/trufflehog:latest \
```

Comment:
Instead of using `latest` we should use an env var for the tag, and manage it with renovate.

e.g. https://github.com/product-os/flowzone/blob/ac19ed86f96d2fcabbe566af89f14f275005e914/flowzone.yml#L560-L562